source/common/PRU-ICSS/PRU-Hands-on-Labs.rst

```diff
@@ -60,7 +60,7 @@ Software
    However, the notation ***<PRU_SW_PATH>*** will be used throughout
    the labs to reference this assumed installation location. If you use
    a different location please modify the steps below.
--  `CCSv6 <http://processors.wiki.ti.com/index.php/Download_CCS#Code_Composer_Studio_Version_6_Downloads>`__
+-  `CCSv6 <ti.com/tool/download/CCSTUDIO>`__
```

Comment:
Keeping in line with the rest of these changes this URL should probably be a full `https://www.ti.com/...` target